assistants/codespace-assistant/assistant/chat.py

```diff
@@ -50,7 +50,7 @@
 assistant_config = BaseModelAssistantConfig(
     AssistantConfigModel,
     additional_templates={
-        "workspace": WorkspaceAssistantConfigModel,
+        "document_workspace": DocumentAssistantConfigModel,
```

Comment:
this change is likely to break existing Workspace Assistant instances. can we just keep it `"workspace"`?